nginx-certbot/nginx.yaml

```
@@ -0,0 +1,2 @@
+exec: /usr/local/bin/startup.sh
+
```

shouldn't we add log directive : to redirect logs to stdout to be able to use redis streaming them 
example: https://github.com/threefoldtech/js-sdk/blob/055ba866bdd21a712629b57acfc0195615b81a13/jumpscale/install/rootfs/etc/zinit/init.yaml#L3
```suggestion
log: stdout
```